api/src/main/java/org/openmrs/module/sync/api/impl/SyncIngestServiceImpl.java

```diff
@@ -565,7 +570,22 @@ else if (!alreadyExists && isDelete) {
                         SyncUtil.setProperty(o, xmlFieldName, null);
                     }
                     else {
-                        SyncUtil.setProperty(o, field, xmlFieldName, xmlFieldValue, xmlFieldType);
+                        if (o instanceof Obs && xmlFieldName.equals("referenceRange")) {
+                            // If an obs comes through with a reference range, and that reference range is not found, ignore that error.
+                            try {
+                                SyncUtil.setProperty(o, field, xmlFieldName, xmlFieldValue, xmlFieldType);
+                            }
+                            catch (Exception e) {
+                                log.warn("Not yet able to set reference range of " + xmlFieldValue + " on obs");
+                            }
+                        }
+                        else {
+                            // If reference range comes through, keep track of what obs that is associated with so that obs can be updated

```

Comment:
nit: maybe this would be clearer if I wasn't reading in a github diff and it had proper formatting, but on first read it seemed like this comment applied to the whole block...